### PR TITLE
capston project

### PR DESCRIPTION
**Pull request by Frelly0**
Accomplishment 
This my capstone project and during making this project i used many technology which are:

- html
- css
- javaScript

### CODE REVIEW

**Review by joel-kalema — COMMENTED** [review]
Hi @Frelly0 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  🛑 

pirates-of-the-caribbean-johnny-depp


- [ ] You have submitted a project that doesn't have a descriptive [README](https://github.com/microverseinc/readme-template) 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: home.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        <div id="cards">
+
+        </div>
+
+        <button class="container" id="join">
+            Get your tickets today!
+        </button>
+
+        <h3>SEE THE WHOLE SCHEDULE</h3>
+    </section>
```
Comment (cynthiainga) [inline home.html:89]: - [ ] Kindly fix the design to match the original design "margin, padding,... 🙃 

Your design

Screenshot 2023-07-04 at 23 55 37

Original design

Screenshot 2023-07-04 at 23 54 51

File: home.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    <section class="container" id="partner">
+        <h3>Partner</h3>
+        <hr>
+        <div class="container">
+            <img src="./Image/partners/mozilla.png" alt="">
+            <img src="./Image/partners/google.png" alt="">
+            <img src="./Image/partners/naver.png" alt="">
+            <img src="./Image/partners/daumkakao.png" alt="">
+            <img src="./Image/partners/airbnb.png" alt="">
```
Comment (cynthiainga) [inline home.html:111]: - [ ] All images should contain `alt` attributes `alt="mozilla",...`, kindly fix that 🙏🏾

File: home.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        <div id="logo">
+            <img src="./Image/logo.png">
+        </div>
+        <div id="rights">
+            <p>
+                2023 Creative Commons. <br>
+                Some rights reserved.
+            </p>
+        </div>
+    </footer>
```
Comment (cynthiainga) [inline home.html:124]: - [ ] Please kindly make sure you use the same logo everywhere, using two different logos on your web give some confusion. Kindly fix that on the About page as well 🙃 
- [ ] Also kindly customize the text in the footer, this website didn’t talk about creative commons. Kindly fix that on the About page as well 😁 
- [ ] Also the image should contain an `alt` attribute, kindly fix that

File: about.html [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    <title>Summit 2020</title>
+    <link rel="icon" type="image/x-icon" href="./Image/hugging.png">
+    <link rel="stylesheet" href="./about.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" rel="stylesheet"/>
+</head>
+<body>
+    <div class="container" id="socials">
+        <ul class="top-bar container">
+            <li class="nav-item"><img src="./Image/facebook.png"></i></li>
+            <li class="nav-item"><img src="./Image/twitter.png"></li>
```
Comment (cynthiainga) [inline about.html:16]: - [ ] All images should contain `alt` attributes `alt="...."`, kindly fix that 🙏🏾

File: about.html [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+                and 2023 tour Logo </h3>
+            </div>
+            <hr>
+            <div>
+                <p>
+                The logo of Rwayana's LA Tour 2023 was decided through fan voting from June 23rd to June 30th.
+                </p>
+            </div>
+            <div>
+                <img src="./Image/hugging.png">
```
Comment (cynthiainga) [inline about.html:96]: - [ ] All images should contain `alt` attributes `alt="...."`, kindly fix that 🙏🏾

File: about.html [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+    
+    <section class="container" id="partner">
+        <h3>Partner</h3>
+        <hr>
+        <div class="container">
+            <img src="./Image/partners/mozilla.png"/>
+            <img src="./Image/partners/google.png"/>
+            <img src="./Image/partners/naver.png"/>
+            <img src="./Image/partners/daumkakao.png"/>
+            <img src="./Image/partners/airbnb.png"/>
```
Comment (cynthiainga) [inline about.html:129]: - [ ] All images should contain `alt` attributes `alt="...."`, kindly fix that 🙏🏾

File: README.md [hunk 7]
```diff
@@ -1 +1,193 @@
...
+I would like to thank Santiago Ruido for the immense support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is under the [Creative Commons Zero v1.0 Universal](./CCO.md) license. Using the design created by [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
```
Comment (cynthiainga) [inline README.md:190]: - [ ] The link that you added for the license `Creative Commons Zero v1.0 Universal` is broken please kindly fix that 🙏🏾 

Screenshot 2023-07-04 at 21 54 00

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Frelly0, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] The website has the `Home` and `About` pages ✔️ 
- [ ] Correct GitHub flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Frelly0 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @Frelly0 ,

Great job so far!  

May I highlight the following: 
- [x] Your  project looks good👏  ✔️ 
- [x] You implemented the requirements of the previous TSE✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request merged**